examples/plot_localizer_analysis_permutation.py

```diff
@@ -0,0 +1,148 @@
+"""
+GLM fitting in fMRI
+===================
+
+Full step-by-step example of fitting a GLM to experimental data realizing
+permutation tests and visualizing the results.
+
+More specifically:
+
+1. A sequence of fMRI volumes are loaded
+2. A design matrix describing all the effects related to the data is computed
+3. a mask of the useful brain volume is computed
+4. A GLM is applied to the dataset (effect/covariance,
+   then contrast estimation)
+5. Permutation tests are performed to obtain uncorrected and corrected p values
+at each voxel for a contrast comparing only two conditions.
+
+"""
+
+print(__doc__)
+
+from os import mkdir, path
+
+import numpy as np
+import pandas as pd
+from nilearn import plotting
+
+from nistats.glm import FirstLevelGLM
+from nistats.design_matrix import make_design_matrix
+from nistats import datasets
+from nistats.permutation_tests import first_level_permutation_test
+from nistats.utils import z_score
+import nibabel as nib
+
+
+n_proc = 6
```

Comment:
should be `n_jobs=1`
(examples should work with n_jobs=1, since people will often run them on a laptop).